### ISSUE TITLE
cloudflare_zero_trust_access_group shows deprecation warning

### ISSUE THREAD

**Issue by omerfarukabaci** [cloudflare/terraform-provider-cloudflare#3733]
### Confirmation

- [X] This is a bug with an existing resource and is not a feature request or enhancement. Feature requests should be submitted with Cloudflare Support or your account team.
- [X] I have searched the issue tracker and my issue isn't already found.
- [X] I have replicated my issue using the latest version of the provider and it is still present.

### Terraform and Cloudflare provider version

**Terraform Version**: `1.7.4`
**Cloudflare Provider Version**: `4.40.0`

### Affected resource(s)

`cloudflare_zero_trust_access_group`

### Terraform configuration files

```hcl
resource "cloudflare_zero_trust_access_group" "my_access_group" {
  name       = "My Access Group"
  account_id = var.account_id

  include {
    login_method = [cloudflare_zero_trust_access_identity_provider.google.id]
  }
}
```

### Link to debug output

Missing

### Expected output

No warnings, since I am already using the most recent resource.

### Actual output

```
╷
│ Warning: Deprecated Resource
│ 
│   with cloudflare_zero_trust_access_group.my_group,
│   on zerotrust_access.tf line 47, in resource "cloudflare_zero_trust_access_group" "my_group":
│   47: resource "cloudflare_zero_trust_access_group" "my_group" {
│ 
│ `cloudflare_access_group` is now deprecated and will be removed in the next major version. Use `cloudflare_zero_trust_access_group`
│ instead.
│ 
│ (and 4 more similar warnings elsewhere)
```

### Steps to reproduce

1. Just try to create a `cloudflare_zero_trust_access_group`, and you will see the warning.

### References

The bug is introduced here: https://github.com/cloudflare/terraform-provider-cloudflare/pull/3584/files#diff-4ae7bd7759a37ef1ead791fcae9d7d3a02156d7f9fd95c7c614f3067e8a132bfR200

**Comment by omerfarukabaci** [comment]
☝🏼 I would like to work on this issue.

**Comment by omerfarukabaci** [comment]
I am closing this issue, it will be fixed with #3740.
